### DISCUSSION TITLE
28. Compiling Foundry Zksync:  I cant create  file zkout

### DISCUSSION THREAD

**Discussion by 48cork** [Cyfrin/foundry-full-course-cu#2582] [General]
after run the command forge build --zksync     the file zksync wasn`t create.   How fix it?

thanks in advance

**Comment by EngrPips** [comment]
Hello @48cork, run `foundryup-zksync` and then retry the building process let's see if that helps.

**Comment by 48cork** [comment]
I got any reply. Doesn`t work

sorry

**Reply by EngrPips** [reply]
I am unsure what could be wrong. Please check on the official `foundry-zksync` repo https://github.com/matter-labs/foundry-zksync If anyone has reported a similar issue and how they solved it.
